elements/rh-site-status/docs/20-guidelines.md

```diff
@@ -0,0 +1,154 @@
+<style>
+  .grid {
+    display: grid;
+    grid-template-columns: 1fr;
+    gap: var(--rh-space-4xl, 64px);
+  }
+
+  .grid .example {
+    width: 100%;
+  }
+
+  .grid .example + .example {
+    margin-top: unset;
+    margin-block-start: unset;
+  }
+
+  .grid.icons .example {
+    margin-block-end: var(--rh-space-xl, 24px);
+  }
+
+  .do {
```

Comment:
Iirc we already have helpers for do and don't

Reply:
There is a different style of this with the red border via `{% example palette="wrong" %}`.  However as I understand it we are moving away from that by way of this this new design that was in the figma design for this page, and this code that as it stands is acting as a temporary setup between this and the larger docs rewrite PR which includes a `uxdot-` component for `do | dont`.

I had mentioned to @coreyvickery while I was working on this that the way I wrote it in this PR will be fairly easy to migrate  the component style when this page ultimately gets rebased into that PR.